agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -229,7 +229,10 @@ private BatchItemProcessor initBatchItemProcessor(
         LazyHttpClient.newHttpPipeLine(
             aadAuthentication,
             new NetworkStatsbeatHttpPipelinePolicy(statsbeatModule.getNetworkStatsbeat()));
-    TelemetryPipeline telemetryPipeline = new TelemetryPipeline(httpPipeline, statsbeatModule);
+    // TODO (heya) refactor the following by using AzureMonitorHelper.createTelemetryItemExporter by
```

Comment:
Will fix the todo when https://github.com/Azure/azure-sdk-for-java/pull/37620 is released.